### PR TITLE
Update fg-delegate.md

### PR DESCRIPTION
**Pull request by kat-enos**
Relnotes for Harness Delegate in SaaS 785xx CG

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the maintainers. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 1]
```diff
@@ -10,6 +10,19 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
...
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
+## February 23, 2023, version 785xx
+### Delegate version 78500
+
+Harness FirstGen release 785xx includes the following feature changes and fixes for Harness Delegate.
+
+### What's new
+
+- The delegate hostname can be used for dynamic selection. You can select a delegate from those in your delegate groups during pipeline runtime. (DEL-5052)
```
Comment (pratmit) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:20]: The delegate hostname can be used for dynamic selection. You can select a delegate hostname from those in your delegate groups during pipeline runtime.

**Review by pratmit — CHANGES_REQUESTED** [review]

File: docs/first-gen/firstgen-release-notes/fg-delegate.md (hunk 1 above)
Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:20]: Same question as the other one.

If you want an alternate, how about:

You can dynamically select delegates by hostname during pipeline runs. To do so, select delegates by hostname from your delegate groups.

Comment (pratmit) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:20]: This works !

Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:20]: I will implement.

Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md:20]: Thanks.

Comment (kat-enos) [reply docs/first-gen/firstgen-release-notes/fg-delegate.md]: Issue resolved. Resolving the conversation.

File: docs/first-gen/firstgen-release-notes/fg-delegate.md [hunk 2]
```diff
@@ -10,6 +10,19 @@ To identify the cluster that hosts your account, open Harness FirstGen and go to
 
 For FirstGen SaaS release notes, see [Harness SaaS Release Notes (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-saa-s-release-notes.md). For Self-Managed Enterprise Edition release notes, see [Self-Managed Enterprise Edition (FirstGen)](/docs/first-gen/firstgen-release-notes/harness-on-prem-release-notes.md).
 
+## February 23, 2023, version 785xx
+### Delegate version 78500
+
+Harness FirstGen release 785xx includes the following feature changes and fixes for Harness Delegate.
```
Comment (aimurphy) [inline docs/first-gen/firstgen-release-notes/fg-delegate.md:16]: version number here too

**Review by aimurphy — APPROVED** [review]
approved w 2 comments about the version number

**Review by aimurphy — APPROVED** [review]

**Comment by kat-enos** [comment]
Thanks April.

**Pull request merged**